Source/WebCore/PAL/pal/graphics/WebGPU/Impl/WebGPUQueueImpl.h

```diff
@@ -83,6 +83,7 @@ class QueueImpl final : public Queue {
         const Extent3D& copySize) final;
 
     void setLabelInternal(const String&) final;
+    void destroy() final { }
```

Comment:
Wait, why does the Impl need destroy() methods? The purpose of the destroy() methods is to remove objects from the WebKit::ObjectHeap, which will cause them to be destroyed; no need for methods on objects to tell them to destroy themselves...

Reply:
I'm confused, why do all these objects have destroy() methods if they don't actually do anything?